virtcontainers/kata_shim.go

```diff
@@ -57,5 +59,17 @@ func (s *kataShim) start(sandbox *Sandbox, params ShimParams) (int, error) {
 		args = append(args, "-log", "debug")
 	}
 
+	if config.Trace {
+		args = append(args, "-trace")
+
+		// FIXME: CRITICAL: should use "DisableNewNetNs" instead
```

Comment:
please file an issue :bug:

Reply:
Good catch! That chunk should have been removed - branch updated!